apps/src/lib/node/ledger/tendermint_node.rs

```diff
@@ -140,6 +155,25 @@ fn kill_on_term_signal(abort_sender: Sender<bool>) {
     });
 }
 
+#[cfg(target_os = "windows")]
+fn kill_on_term_signal(abort_sender: Sender<bool>) {
+    use tokio::signal::ctrl_c;
+    let _ = std::thread::spawn(|| {
+        tokio_test::block_on(async move {
```

Comment:
`tokio_test::block_on` creates a new runtime, we already have one in the shell on `run_shell`. I think we'll want to refactor the ledger to have the main entrypoint async so that this function can be async too